uat/testing-features/src/main/resources/greengrass/features/log-manager-1.feature

```diff
@@ -9,6 +9,24 @@ Feature: Greengrass V2 LogManager
         And 5 temporary rotated log files for component aws.greengrass.Nucleus have been created
         And 5 temporary rotated log files for component UserComponentA have been created
 
+    @local
+    Scenario: LogManager-1-T0: install the local custom component
+        Given I create a Greengrass deployment with components
+            | aws.greengrass.Cli        | LATEST |
+        And I deploy the Greengrass deployment configuration
+        Then I wait 120 seconds
+        Then the Greengrass deployment is COMPLETED on the device after 2 minutes
+        When I install the component logGenerator from local store with configuration
+        """
+        {
+            "MERGE": {
+                "FileSize": "10MB",
```

Comment:
Maybe we should split the unit and size so parsing becomes easier. Also speed might not be the appropriate wording in this context (speed = distance traveled / time elapsed) and we are going to use it to how frequent we write a log into a file. Maybe try



```suggestion
                "FileSize": "10",
                "FileSizeUnit": "MB",
                "LogWriteFrequencySeconds": 1
```

Reply:
This is just temporary for testing if the current toy component can receive parameters. They should be well-defined once the custom component is developed.

Reply:
Why even add any value then?

Reply:
Good suggestion. I will apply them when developing custom component